<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>1009</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Косарев А. ИУ7-51Б</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imgs/icon.ico</normaloff>imgs/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SpecialGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>1541</width>
      <height>951</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>1281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Конструктор материнских плат</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGPU">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GPU</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left_scene">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCreateMotherboard">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Построить плату</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProcessor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавить процессор</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rightCircle">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↷</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Вращение платы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Перемещение платы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_distance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Масштабирование платы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddGPU">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>820</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить блок видеопамяти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddProcessor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить процессор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sceneToInitianPosition">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вид сверху</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_leftCircle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>↶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deleteModel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>940</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить элемент</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRAM">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RAM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddLight">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>900</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить источник света</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMotherboardType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetRAM">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Меню управления</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddRAM">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить блок RAM памяти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_zoom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_right_scene">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_left">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetGPU">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_down_scene">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetProcessor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up_scene">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_up">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGPUColor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>850</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Менять цвет видеокарт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeRAMColor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>710</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Менять цвет блоков RAM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonChangeCPUColor">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Менять цвет процессора</string>
    </property>
   </widget>
   <zorder>pushButton_up</zorder>
   <zorder>graphicsView</zorder>
   <zorder>label_6</zorder>
   <zorder>labelGPU</zorder>
   <zorder>pushButton_left_scene</zorder>
   <zorder>pushButtonCreateMotherboard</zorder>
   <zorder>labelProcessor</zorder>
   <zorder>pushButton_rightCircle</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_right</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_distance</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButtonAddGPU</zorder>
   <zorder>pushButtonAddProcessor</zorder>
   <zorder>pushButton_sceneToInitianPosition</zorder>
   <zorder>pushButton_leftCircle</zorder>
   <zorder>pushButton_deleteModel</zorder>
   <zorder>labelRAM</zorder>
   <zorder>pushButtonAddLight</zorder>
   <zorder>pushButton_down</zorder>
   <zorder>comboBoxMotherboardType</zorder>
   <zorder>listWidgetRAM</zorder>
   <zorder>label</zorder>
   <zorder>pushButtonAddRAM</zorder>
   <zorder>pushButton_zoom</zorder>
   <zorder>pushButton_right_scene</zorder>
   <zorder>pushButton_left</zorder>
   <zorder>listWidgetGPU</zorder>
   <zorder>pushButton_down_scene</zorder>
   <zorder>listWidgetProcessor</zorder>
   <zorder>pushButton_up_scene</zorder>
   <zorder>pushButtonGPUColor</zorder>
   <zorder>pushButtonChangeRAMColor</zorder>
   <zorder>pushButtonChangeCPUColor</zorder>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>Руководство пользователя</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpecialGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>specialgraphicsview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
